packages/asc-ui/src/components/SearchBar/SearchBar.stories.tsx

```diff
@@ -115,7 +105,33 @@ const SearchBarMobileStory: React.FC<{}> = () => {
   )
 }
 
+const SearchBarMenuStory: React.FC<{}> = () => {
+  const [searchText, setText] = React.useState('')
+
+  return (
+    <>
+      <SearchBarMenu
+        placeholder="Enter the search text"
+        onTextChanged={(text: string) => {
+          setText(text)
+          action(`text changed: ${searchText}`)
+        }}
+        onSearch={() => {
+          action(`button clicked: ${searchText}`)
+        }}
+        text={searchText}
+      />
+
+      <p>
+        <span>Searching for:</span>
+        {searchText}
+      </p>
+    </>
+  )
+}
+
 storiesOf('Composed/SearchBar', module)
   .add('default state', () => <SearchBarStory />)
   .add('with changed style', () => <SearchBarStoryWithChangedStyle />)
-  .add('mobile with changed style', () => <SearchBarMobileStory />)
+  .add('mobile with changed style', () => <ChangedSearchBarMenuStory />)
```

Comment:
What exactly is the purpose of showing a ChangedSearchBar? Perhaps it's better to document how the props work instead? https://storybook.js.org/docs/configurations/options-parameter/ Because how you should use the passing prop onTextChanged is up to the user basically. :)